contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcMeasureConstants.java

```diff
@@ -212,7 +349,9 @@
    * {@link Measure} for gRPC server uncompressed request bytes.
    *
    * @since 0.8
+   * @deprecated in favor of {@link #GRPC_SERVER_SENT_BYTES_PER_RPC}.
```

Comment:
Same here, I think GRPC_SERVER_SENT_BYTES_PER_RPC and GRPC_SERVER_RECEIVED_BYTES_PER_RPC are swapped in these two Javadocs.